reference/docs-conceptual/Crescendo/advanced/using-crescendo-cmdlets.md

```diff
@@ -194,10 +191,10 @@ $handler.Handler = 'ParseShadow'
 $newCommand.OutputHandlers += $handler
 
 ## Add the command to the Commands collection of the configuration
-$NewConfiguration.Commands += $newCommand
+$CrescendoCommands += $newCommand
 
 # Export the configuration to a JSON file and create the module
-$NewConfiguration | ConvertTo-Json -Depth 5 | Out-File .\vssadmin.json -Force
+Export-CrescendoCommand -command $CrescendoCommands -fileName .\vssadmin.json
```

Comment:
```suggestion
Export-CrescendoCommand -Command $CrescendoCommands -FileName .\vssadmin.json
```

Reply:
The case is correct for how they are defined in Crescendo.